examples/server.config.example

```diff
@@ -39,6 +39,19 @@ instances {
 
     # limit for CAS total content size in bytes
     cas_max_size_bytes: 1073741824 # 1024 * 1024 * 1024
+
+    # an imposed action-key-invariant timeout used in the unspecified timeout case
+    default_action_timeout: {
+      seconds: 600
+      nanos: 0
+    }
+
+    # a limit on the action timeout specified in the action, above which
+    # the operation will report a failed result immediately
```

Comment:
nit: will report a timeout.

Timeout is a special kind of failure which is propagated differently throughout the API.

Reply:
I'm actually set up right now in `MemoryInstance#validateAction` to fail the execute with a Precondition violation if the requested timeout is above the `maximum_action_timeout`, so the execute will fail with FAILED_PRECONDITION (with updates to #73)

I suppose I would want to see something indicating that this should not be the reaction to such an improperly defined action - it does fly somewhat in the face of the specification comments indicating that 'if you fix the precondition state, the action can be retried', since all other instances of that are data consistency issues (missing blobs, directories, etc)